### PR TITLE
HBGM Order - 456

### PR DESCRIPTION
**Pull request by hossenlopp**

### CODE REVIEW

**Comment by yuliumitre** [comment]
Regarding the diagnosis section of the questionnaire, I have question and will send the questions to the team to discuss. Although the paper template only has a checkbox for Diabetes Insulin treated or not, but to be consistent with other forms, we should display the specific diagnoses and provide drop down list from the valueset for the provider to select additional ones. It also misses "Other diagnoses" section.

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json [hunk 1]
```diff
@@ -88,6 +87,11 @@
           ],
           "repeats": "true",
           "answerOption": []
+        },
+        {
```
Comment (yunwwang) [inline HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json]: Remove describe item

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".OtherDiagnosesCodings"
+              }
+            }
+          ],
+          "repeats": "true",
+          "answerOption": []
+        },
+        {
```
Comment (yunwwang) [inline HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json:107]: Remove describe item

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+      "text": "Frequency of Use",
+      "item": [
+        {
+          "type": "group",
+          "linkId": "6.1",
+          "text": "Non-insulin Treated Frequency",
+          "enableWhen": [
+            {
+              "question": "4.2",
+              "operator": "!=",
```
Comment (yunwwang) [inline HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json:277]: Need LHC form 25.1.2 (or later) to work

Comment (hossenlopp) [reply HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json:277]: That was the intention. Perhaps we should get a PR into DTR before this is merged.

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json [hunk 4]
```diff
@@ -80,9 +80,8 @@
       "item": [
         {
           "linkId": "4.1",
-          "text": "Diabetes Mellitus",
+          "text": "Diabetes Mellitus Diagnoses",
```
Comment (hossenlopp) [inline HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json:61]: This PR was supposed to focus on the HBGM Order. Why are there changes to the F2F questionnaire here, @yuliumitre?

Comment (yuliumitre) [reply HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json:61]: This change is triggered by the discussion about "Diagnoses". Original thoughts was to change HBGM order and F2F in this branch as a model. This change just makes the names consistent across order and F2F form.  I also have another branch to address the issues for other topics.

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json (hunk 3 above)
Comment (yunwwang) [reply HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json:277]: Can you create PR for that?

Comment (hossenlopp) [reply HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json:277]: Will do.

Comment (hossenlopp) [reply HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json:277]: PR Created. https://github.com/HL7-DaVinci/dtr/pull/106

**Review by yunwwang — APPROVED** [review]
Looks good

**Pull request merged**